test/test_images/target/main.go

```diff
@@ -25,56 +26,68 @@ func main() {
 		panic(err)
 	}
 
-	fmt.Printf("Target prefix to match: %q.\n", r.Target)
+
```

Comment:
For this part, we have not used the framework used as that in other xxx_target, will change it later